docker/build-tools/Dockerfile

```diff
@@ -393,11 +393,10 @@ ENV DEBIAN_FRONTEND=noninteractive
 
 # Pinned versions of stuff we pull in
 ENV AWESOMEBOT_VERSION=1.20.0
-# Sometime between 1.14.1 and 1.14.2 to pull in some fixes for deb
-ENV FPM_VERSION=eb5370d16e361db3f1425f8c898bafe7f3c66869
-ENV HTMLPROOFER_VERSION=3.19.0
-ENV NOKOGIRI_VERSION=1.14.3
-ENV LICENSEE_VERSION=9.15.1
+ENV FPM_VERSION=1.15.1
+ENV HTMLPROOFER_VERSION=5.0.7
+ENV NOKOGIRI_VERSION=1.15.2
```

Comment:
I will need to look, but I think nokogiri might not be used anymore. I think it was part of the linting on istio.io, but was removed from the linter used there. See htmlproofer above.

Reply:
Note that overall this change will stop us from doing html parsing of istio.io (ie, many lint errors previously caught will not be anymore).

Reply:
I reverted html proofer from 5.x to 3.19.4. We shouldn't remove html parsing. What should I change in the PR?

Reply:
What I would suggest doing is building a test image and running make lint against istio.io to verify the NOKOGIRL version is acceptable as well. I specifically had to pin the version to 1.14.3 a few weeks back because 1.15.0? caused the linting to fail. Just want to make sure that is fixed before moving up to a later 1.15.x.

Reply:
https://github.com/istio/tools/pull/2469 was my earlier change